src/views/GovernanceTaskList/index.tsx

```diff
@@ -1,377 +1,217 @@
+                      <div className="margin-top-2">
+                        <UswdsReactLink to="./">
+                          {t('taskList.steps.3.link')}
+                        </UswdsReactLink>
+                      </div>
+                    </TaskListDescription>
+                  </TaskListItem>
+
+                  {/* 5. Submit your Business Case for final approval */}
+                  <TaskListItem
+                    heading={t('taskList.steps.4.title')}
+                    status={itGovTaskStatuses.bizCaseFinalStatus}
+                    testId={kebabCase(t('taskList.steps.4.title'))}
+                  >
+                    <TaskListDescription>
+                      <p>{t('taskList.steps.4.description')}</p>
+                    </TaskListDescription>
+                  </TaskListItem>
+
+                  {/* 6. Attend the GRB meeting */}
+                  <TaskListItem
+                    heading={t('taskList.steps.5.title')}
+                    status={itGovTaskStatuses.grbMeetingStatus}
+                    testId={kebabCase(t('taskList.steps.5.title'))}
+                  >
+                    <TaskListDescription>
+                      <p>{t('taskList.steps.5.description')}</p>
+                      <div className="margin-top-2">
+                        <UswdsReactLink to="./">
+                          {t('taskList.steps.5.link')}
+                        </UswdsReactLink>
+                      </div>
+                    </TaskListDescription>
+                  </TaskListItem>
+
+                  {/* 7. Decision and next steps */}
+                  <TaskListItem
+                    heading={t('taskList.steps.6.title')}
+                    status={itGovTaskStatuses.decisionAndNextStepsStatus}
+                    testId={kebabCase(t('taskList.steps.6.title'))}
+                  >
+                    <TaskListDescription>
+                      <p>{t('taskList.steps.6.description')}</p>
+                    </TaskListDescription>
+                  </TaskListItem>
+                </TaskListContainer>
+              </div>
+            </Grid>
+
+            {/* Sidebar */}
+            <Grid tablet={{ col: 3 }}>
+              <div className="line-height-body-4 padding-top-3 border-top border-top-width-05 border-primary-lighter">
+                <div>
+                  <UswdsReactLink to="/system/making-a-request">
+                    {t('button.saveAndExit')}
+                  </UswdsReactLink>
+                </div>
+                <div className="margin-top-1">
+                  <Button type="button" unstyled className="text-error">
+                    {t('button.removeYourRequest')}
+                  </Button>
+                </div>
+
+                <h4 className="line-height-body-2 margin-top-3 margin-bottom-1">
+                  {t('taskList.help')}
+                </h4>
+                <div className="margin-top-1">
```

Comment:
Just a suggestion, but if you add `display-block` to link elements you can add the margin utility classes there instead of wrapping all the links in extra divs.

Reply:
I like the thought but after trying it out I think I should just leave the divs/ block wrappers.

Text links narrower than the container end up with a hit box too big.

<img width="232" alt="Screen Shot 2023-07-03 at 1 51 36 PM" src="https://github.com/CMSgov/easi-app/assets/97050498/c74efd1c-7510-4e26-bdf0-13591a6c0804">

Reply:
My bad, I meant `display-inline-block` to keep the element the same size. But I'll go ahead and approve.

Reply:
Np. I just tried that again with inline block to be sure. The effect now is that I have to somehow clear the neighboring inline elements, these buttons and links can be right next to each other. I'm not aware of any util classes that do this, I just see `.clearfix` which is for something different. I'm not quick enough with css atm to know the rule to get this done as short as just block wrapping with divs.